bitnami/nginx-ingress-controller/Chart.yaml

```diff
@@ -34,4 +34,4 @@ maintainers:
 name: nginx-ingress-controller
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/nginx-ingress-controller
-version: 9.8.1
+version: 9.8.2
```

Comment:
Could you please remove this change? Now the execution succeeded, it's no longer needed 😁

Reply:
done at https://github.com/bitnami/charts/pull/19139/commits/b8582ced97b346b71141fc18fc00f6ce9debe8d9